### PR TITLE
[Session] Initialize the default locale in the constructor

### PR DESCRIPTION
**Pull request by tacman**
also, remove the $ so gitclip works.

### CODE REVIEW

File: session.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ You need to install the HttpFoundation component to handle sessions:
 
 .. code-block:: terminal
 
-    $ composer require symfony/http-foundation
+    composer require symfony/http-foundation
```
Comment (OskarStark) [inline session.rst:29]: Please revert this, maybe @javiereguiluz can modify the copy button for this special case?

I also added a rule to DOCtor-RST the we ensure to keep this `$`-prompt for composer commands.

https://github.com/OskarStark/doctor-rst/pull/1307

Comment (javiereguiluz) [reply session.rst:29]: But that's already the case. You can't select this prompt. See:

text-selection-prompt

Comment (OskarStark) [reply session.rst:29]: Ah I thought we have a copy button next to it on the right side

Comment (javiereguiluz) [reply session.rst:29]: On the pages where we show a "Copy" button (e.g. https://symfony.com/download) we also remove the prompt from the copied contents.

**Comment by javiereguiluz** [comment]
I'm sorry but I must close this as "won't merge" because this change was already made in a different PR. See the changed constructor argument in https://github.com/symfony/symfony-docs/blob/6.2/session.rst#creating-a-localesubscriber

We hope to merge your future pull requests. Thanks!

**Pull request closed without merging**